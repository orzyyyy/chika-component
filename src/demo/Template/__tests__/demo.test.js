import React from 'react';
import { shallow } from 'enzyme';
import TemplateDemo from '..';

describe('TemplateDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<TemplateDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
