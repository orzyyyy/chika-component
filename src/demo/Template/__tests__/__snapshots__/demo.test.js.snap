// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemplateDemo render correctly 1`] = `
<Template
  dataSource={
    Array [
      Object {
        "id": "template-1",
        "test1": "test1",
        "test2": "test2",
      },
      Object {
        "id": "template-2",
        "test1": "test11",
        "test2": "test22",
      },
    ]
  }
>
  <div
    data-key="test1"
  />
  <div
    data-key="test2"
  />
</Template>
`;
